acceptance/setup/pre_suite/15_setup_repos.rb

```diff
@@ -22,7 +26,12 @@ def initialize_repo_on_host(host, os)
 enabled=1
 gpgcheck=1
     REPO
+    end
 
+    host['platform'].with_version_codename =~ /^(fedora|el|centos)-(\d+)-(.+)$/
+    variant = (($1 == 'centos') ? 'el' : $1)
+    version = $2
+    unless version == '7'
```

Comment:
In ec2 or vmware? This is definitely a case of "Works For Me" at the moment, with automated tests to back this up: https://jenkins.puppetlabs.com/view/PuppetDB/view/master%20-%20Ezbake/job/platform_puppetdb_intn-sys/314/BEAKER_CONFIG=ec2-west-el7-64mda-el7-64a,BEAKER_OPTIONS=postgres,PUPPETDB_INSTALL_MODE=install,label=beaker-ec2/consoleFull ... so ... we should figure out why. Perhaps its environmental differences.